src/tests/JIT/Regression/JitBlue/Runtime_98204/Runtime_98204.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+public static class Runtime_98068
+{
+    [Fact]
+    [InlineData(1e18)]
```

Comment:
Can this be added to the existing SinCos test under libraries instead of creating a whole new core runtime test for it?

Reply:
Fixed.